src/App.js

```diff
@@ -4,9 +4,46 @@ import { Route, Link, Switch } from "react-router-dom";
 import { AppBar, Typography, Toolbar, Button} from "@material-ui/core";
```

Comment:
Import each component separately like this:

```
import AppBar from "@material-ui/core/AppBar";
import Typography from "@material-ui/core/Typography";
import Toolbar from "@material-ui/core/Toolbar";
import Button from "@material-ui/core/Button";
```

This will reduce the overall bundle size when we compile the frontend :)